topic3/src/keyPoint2/WordWrapper.java

```diff
@@ -0,0 +1,66 @@
+package keyPoint2;
+
+import java.util.Spliterator;
+
+public class WordWrapper {
+
+	private int rowLength;
+	private String word;
+
+	public WordWrapper(int rowLength, String word) {
+		this.rowLength = rowLength;
+		this.word = word;
+	}
+
+	public void print() {
+
+		switch (rowLength) {
```

Comment:
60, 7, 3 and 5 are just examples. Consider that rowLength can be any integer between 1 and 100